### PR TITLE
Use ongoing projects for global project Endpoint

### PR DESCRIPTION
**Pull request by thenav56**
Addresses  3W: Global view: Create Aggregated API #1092  
cc: @frozenhelium 

## Changes

* Only use ongoing projects.

### CODE REVIEW

**Review by vdeak — APPROVED** [review]

File: deployments/drf_views.py [hunk 1]
```diff
@@ -406,7 +406,13 @@ def get_projects(self):
         # Filter by visibility
         qs = self.get_visibility_queryset(Project.objects.all())
         # Filter by GET params
-        return ProjectFilter(self.request.query_params, queryset=qs).qs
+        projects = ProjectFilter(self.request.query_params, queryset=qs).qs
+        return Project.objects.filter(
```
Comment (batpad) [inline deployments/drf_views.py:410]: I'm sure there's a good reason, but why not just `project.objects.filter(...)` here and why do we need the `id__in=projects` here? Just curious.

**Review by batpad — APPROVED** [review]

File: deployments/drf_views.py (hunk 1 above)
Comment (thenav56) [reply deployments/drf_views.py:410]: I am not sure, but I remember getting duplicate rows and wasn't able to use distinct() here. So to be sure, I am passing the filtered queryset as a subquery.

Comment (batpad) [reply deployments/drf_views.py:410]: I see, could we add a comment to that effect?

Strange that distinct doesn't work, but I know I've had the problem where it's some-how really hard to get rid of duplicates with complex joins - have you tried something like `.distinct('id')` or so? Anyways, this looks fine - might just be nice to have a comment.

Comment (thenav56) [reply deployments/drf_views.py:410]: Sorry, but there is a comment. Do you want me to extend it?

Comment (batpad) [reply deployments/drf_views.py:410]: Sorry, my bad - no, this is fine. Shall I go ahead and merge this?

Comment (thenav56) [reply deployments/drf_views.py:410]: Yes please merge and deploy to staging.

**Comment by frozenhelium** [comment]
@vdeak Can you please deploy this to staging?

**Comment by vdeak** [comment]
@frozenhelium It is deployed to staging

**Comment by frozenhelium** [comment]
Thanks :)

**Pull request merged**